generators/search-map.js

```diff
@@ -25,17 +25,19 @@ module.exports = function(docMap, siteConfig) {
 	for (name in docMap) {
 		if (docMap.hasOwnProperty(name)) {
 			var docObj = docMap[name];
-			
-			var description = helpers.stripMarkdown(docObj.description);
-			description = unescapeHTML(description);
+			// If there is no body, it's likely we don't want to index it
+			if(docObj.description || docObj.signatures || docObj.params || docObj.return || docObj.options){
```

Comment:
When I look at the `docObject` that’s on `window` on any of the API pages, there are `description` and `signatures` properties but not the others (they’re defined in `signatures`). Am I looking at the wrong thing to verify this code?